tasks/rslang/english-for-kids-translated.md

```diff
@@ -0,0 +1,153 @@
+# English for kids
+
+| Deadline         | Folder name| Branch name |
+| ---------------- | ----------- | ----------- |
+| 19.04.2020 23:59 | english-for-kids    | english-for-kids    |
+
+**English for kids** - an application for learning English words for kids.
+
+## Application Structure:
+1. Main page
+2. Category Page
+3. Statistics Page
+
+![screenshot](english-for.kids.data/img/screenshot.png)
+### Demo
+https://english-for-kids.netlify.com/ (version without Hacker scope)
+
+## Examples of work
+- https://marta-r-english-for-kids.netlify.app/
+- https://zheromskyv-english-for-kids.netlify.app/
+- https://xmelsky-english-for-kids.netlify.app/
+- https://irinateln0va-english-for-kids.netlify.app/
+- https://kotiknalune-english-for-kids.netlify.app/
+
+## Description of Pages
+1. The Main Page of the Application
+- there are links to the categories pages of words on the main page of the application
+- the minimum number of categories is eight
+- each link contains a thematic image and a category name
+- links are duplicated in the animated sidebar menu, which appears and disappears by toggling the icon in the upper left corner of the page
+- on the main page and category pages of the application there is a Train / Play switch button
+
+2. Category Page
+- category page includes category name and word cards related to category theme
+- the minimum number of word cards in each category is eight
+- each card has a thematic picture and an english word
+- when you click on the card, you will hear the the pronunciation of the word in English
+- each card has a button on the right bottom corner, when you click on that button the card flips over. The back side of the card has a translation of the word. Clicking on the back side of the card does not trigger any events like pronunciation of the word.
+- card is rotated back to the front side when the mouse cursor moves beyond the borders of the card
+
+3. Statistics Page
+- the page description is added in the evaluation criteria (Hacker scope)
+
+## How Application should work
+The application works in training and game modes.
+Description of the application in these modes is added in evaluation criteria (Basic scope and Advanced scope, respectively). When you download the application or reload the page, the application opens in training mode. Switching between training and playing modes is triggered by toggling the Train / Play switch button.
+
+## Repository requirements
```

Comment:
The links to the pages aren't translated into English. In addition, I'm not sure about the accuracy of the Repository requirements described here (e.g. in which repository the work is to be done).
You can refer to the technical requirements here:
https://github.com/rolling-scopes-school/tasks/blob/master/tasks/virtual-keyboard/virtual-keyboard-en.md#requirements-for-commits-pull-request-repository

@Anik188 Could you please review the requirements?